File: src/main/java/com/chbinou/smarthouse/app/config/Constantes.java
package com.chbinou.smarthouse.app.config;

/**
 * Created by nxuser on 07/01/2017.
 */
public class Constantes
{
    public interface Url
    {
        String ANY                              = "/*";
        String API_PUSH_WSOCKET                 = "/push";
        String PATH_API                         = "/api";

        String LOGIN                            = "/login";
        //Lamps
        String PATH_SWITCHING                   = "/switching";
        String PATH_LAMP                        = "/lamp";
        String API_STATUS_LAMP_ALL              = "/all/status";
        String API_SWITCHON_LAMP                = "/:identifierzone/:identifier/on";
        String API_SWITCHOFF_LAMP               = "/:identifierzone/:identifier/off";
        String API_SWITCHON_LAMP_ALL            = "/all/on";
        String API_SWITCHOFF_LAMP_ALL           = "/all/off";
        //climatiseurs
        String PATH_COOLER                      = "/climatiseur";
        String API_SWITCHON_CLIMATISEUR         = "/:identifierzone/:identifier/on";
        String API_SWITCHOFF_CLIMATISEUR        = "/:identifierzone/:identifier/off";
        String API_SWITCHON_CLIMATISEUR_ALL     = "/all/on";
        String API_SWITCHOFF_CLIMATISEUR_ALL    = "/all/off";
        //Windows
        String PATH_WINDOW                      = "/window";
        String API_SWITCHUP_WINDOW              = "/:identifierzone/:identifier/up";
        String API_SWITCHDOWN_WINDOW            = "/:identifierzone/:identifier/down";
        String API_SWITCHUP_WINDOW_ALL          = "/all/up";
        String API_SWITCHDOWN_WINDOW_ALL        = "/all/down";
        String API_POSITION_WINDOW              = "/:identifierzone/:identifier/:pos";
        String API_POSITION_WINDOW_ALL          = "/all/:pos";




    }
}

File: backend/api/config/Constantes.py
ANY                              = "/*";
API_PUSH_WSOCKET                 = "/push";
PATH_API                         = "/api";

LOGIN                            = "/login"
#Lamps
PATH_SWITCHING                   = "/switching"
PATH_LAMP                        = "/lamp"
API_STATUS_LAMP_ALL              = "/api/switching/lamp/all/status"
API_SWITCHON_LAMP                = "/api/switching/lamp/<identifierzone>/<identifier>/on"
API_SWITCHOFF_LAMP               = "/api/switching/lamp/<identifierzone>/<identifier>/off"
API_SWITCHON_LAMP_ALL            = "/api/switching/lamp/all/on"
API_SWITCHOFF_LAMP_ALL           = "/api/switching/lamp/all/off"
#climatiseurs
PATH_COOLER                      = "/api/switching/climatiseur"
API_SWITCHON_CLIMATISEUR         = "/api/switching/climatiseur/<identifierzone>/<identifier>/on"
API_SWITCHOFF_CLIMATISEUR        = "/api/switching/climatiseur/<identifierzone>/<identifier>/off"
API_SWITCHON_CLIMATISEUR_ALL     = "/api/switching/climatiseur/all/on"
API_SWITCHOFF_CLIMATISEUR_ALL    = "/api/switching/climatiseur/all/off"

#Windows
PATH_WINDOW                      = "/api/switching/window"
API_SWITCHUP_WINDOW              = "/api/switching/window/<identifierzone>/<identifier>/up";
API_SWITCHDOWN_WINDOW            = "/api/switching/window/<identifierzone>/<identifier>/down";
API_SWITCHUP_WINDOW_ALL          = "/api/switching/window/all/up"
API_SWITCHDOWN_WINDOW_ALL        = "/api/switching/window/all/down"
API_POSITION_WINDOW              = "/api/switching/window/<identifierzone>/<identifier>/<pos>";
API_POSITION_WINDOW_ALL          = "/all/<pos>"

